Gesture_control.py

```diff
@@ -8,6 +8,34 @@
 import cv2
 import numpy as np
 
+
+def get_trackbars():
+    cv2.namedWindow('Trackbars')
+    cv2.resizeWindow('Trackbars', 640, 240)
+    cv2.createTrackbar('HueMin', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('HueMax', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('SatMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('SatMax', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMax', 'Trackbars', 0, 255, empty)
+
+def create_mask():
+    while True:
+        img = cv2.imread('') # read the current frame
```

Comment:
You have to apply this function to the input frame of the video. So, no need to take this img. You can take img as a parameter in this function.